### PR TITLE
#2978. show tab with arrow to bar

### PR DESCRIPTION
**Pull request by maxyeg**
Remind please, that in our code cells when we press Tab key it adds two spaces, not `\t`.

### CODE REVIEW

**Comment by maxyeg** [comment]
issues #2969 #2978

File: core/src/main/web/app/mainapp/components/notebook/codecell-directive.js [hunk 1]
```diff
@@ -379,6 +379,7 @@
           scope.cm = CodeMirror.fromTextArea(element.find('textarea')[0], codeMirrorOptions);
           scope.bkNotebook.registerCM(scope.cellmodel.id, scope.cm);
           scope.cm.on('change', changeHandler);
+          scope.cm.on('change', bkCoreManager.replaceTabsWithArrow);
```
Comment (tomca32) [inline core/src/main/web/app/mainapp/components/notebook/codecell-directive.js:382]: Maybe we can debounce this. If you're running this on every keystroke with a fast typist we could run into problems.

We use lodash, lodash has debounce: https://lodash.com/docs#debounce

**Comment by scottdraves** [comment]
this is what i want actually, not to actually replace the tabs but to show them differently only:
https://codemirror.net/demo/visibletabs.html

**Comment by maxyeg** [comment]
That's not what we need

**Pull request closed without merging**